libs/shared/utils/src/lib/getOrganizationLogoUrl.ts

```diff
@@ -28,5 +18,3 @@ const getOrganizationLogoUrl = (
   }
   return ''
```

Comment:
Do you know why this utility returns "" instead of the fallback image if there are 0 orgs? I think that's a bad fallback since it might cause extra requests to the HTML page.

Reply:
I am not sure why it was implemented this way. But good catch! :) I will change it to the fallback image.